app/models/user.rb

```diff
@@ -5,7 +5,7 @@ class User < ApplicationRecord
   include PgSearch::Model
   include RecruitmentCycleHelper
 
-  before_save :downcase_email
+  before_save :downcase_email, :strip_email_whitespace
```

Comment:
```suggestion
  normalizes :email, with: -> email { email.strip.downcase }
```

That should work without the `before_save` hook, according the documentation.

Reply:
This only possible in rails 7.1 **or** with the normalize gem. I don't think it's worth adding the gem just for this. 

https://www.shakacode.com/blog/rails-add-normalize-to-activerecord/#:~:text=Normalization%20is%20applied%20when%20the,subsequent%20querying%20using%20unnormalized%20values.

Reply:
Ooh right! I hadn't realised we weren't on Rails 7 yet. Sorry about that!

Reply:
No probs, we need to do the upgrade!